scripts/o3de/ExportScripts/export_script_only_project.py

```diff
@@ -0,0 +1,270 @@
+                            help="Option to create a compressed archive the output. "
+                                 "Specify the format of archive to create from the output directory. If 'none' specified, no archiving will occur.",
+                            choices=["none", "zip", "gzip", "bz2", "xz"], default="none")
+        parser.add_argument('-assets', '--should-build-assets', default=False, action='store_true',
+                            help='Toggles building all assets for the bundle.')
+        parser.add_argument('-foa', '--fail-on-asset-errors', default=False, action='store_true',
+                            help='Option to fail the project export process on any failed asset during asset building (applicable if --should-build-assets is true)')
+        parser.add_argument('-sl', '--seedlist', type=pathlib.Path, dest='seedlist_paths', action='append',
+                                help='Path to a seed list file for asset bundling. Specify multiple times for each seed list.')
+        parser.add_argument('-gpfp', '--game-project-file-pattern-to-copy', type=str, dest='game_project_file_patterns_to_copy', action='append',
+                                help="Any additional file patterns located in the project directory. File patterns will be relative to the project path. Will be included in GameLauncher.")
+        parser.add_argument('-spfp', '--server-project-file-pattern-to-copy', type=str, dest='server_project_file_patterns_to_copy', action='append',
+                                help="Any additional file patterns located in the project directory. File patterns will be relative to the project path. Will be included in ServerLauncher.")
+        parser.add_argument('-pfp', '--project-file-pattern-to-copy', type=str, dest='project_file_patterns_to_copy', action='append',
+                                help="Any additional file patterns located in the project directory. File patterns will be relative to the project path.")
+        parser.add_argument('-lbp', '--launcher-build-path', type=pathlib.Path, default=None,
+                            help="Designates where the launcher build files (Game/Server/Unified) are generated. If not specified, default is <o3de_project_path>/build/launcher.")
+        parser.add_argument('-regovr', '--allow-registry-overrides', default=False, type = bool,
+                            help="When configuring cmake builds, this determines if the script allows for overriding registry settings from external sources.")
+        parser.add_argument('-abp', '--asset-bundling-path', type=pathlib.Path, default=None,
+                            help="Designates where the artifacts from the asset bundling process will be written to before creation of the package. If not specified, default is <o3de_project_path>/build/asset_bundling.")
+        parser.add_argument('-maxsize', '--max-bundle-size', type=int, default=2048, help='Specify the maximum size of a given asset bundle.')
```

Comment:
Could you add to the help message that this is the maximum bundle size in Mebibytes?
```suggestion
        parser.add_argument('-maxsize', '--max-bundle-size', type=int, default=2048, help='Specify the maximum size of a given asset bundle in MiB.')
```

Reply:
Sure, note that this was copied from the other script in the same folder, so I would probably modify both.